src/inmanta/data/__init__.py

```diff
@@ -2196,26 +2196,28 @@ def to_dto(self) -> m.CompileRun:
 
 class LogLine(DataDocument):
```

Comment:
This could use a type annotation for the level field.

Reply:
I don't understand this comment.  There is no `level` field? There is a `level` attribute in the `_data` dict, is it what you are referring to? Or is it the property?  In the former case, how can I add any type annotation to this?  In the later, it is done already no?

Reply:
Ah yes, I'm talking about the`level` in `_data`. I would add comments to document what is in data.

Reply:
Done in https://github.com/inmanta/inmanta-core/pull/3199/commits/ffb53a8cfda2a0bc7f1b17dc49162e3d69f918af

Reply:
nice!